app/src/main/java/com/group80/uoftinder/RecommendationView.java

```diff
@@ -0,0 +1,112 @@
+package com.group80.uoftinder;
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+/**
+ * Public class that extends AppCompatActivity and implements RecViewInterface.
+ * This class displays the most compatible users to currentUser.
+ */
+public class RecommendationView extends AppCompatActivity implements RecViewInterface {
+    private final User currentUser;
+    private final RecommendationPresenter recPresenter;
+    private User displayedUser;
+
+    /**
+     * Initialize an instance of RecommendationView by passing the currentUser.
+     * @param currentUser is the current user.
+     */
+    public RecommendationView(User currentUser) {
+        this.currentUser = currentUser;
+        this.recPresenter = new RecommendationPresenter(this);
```

Comment:
I don't think `this` is necessary here? `RecommendationPresenter` does not have a constructor with an argument as of the latest changes in #22.

Reply:
I am very sorry for not committing this yet but I will do so shortly. I realized that RecommendationPresenter requires an instance of the View, which is not created anywhere. Thus, I added a parameter into RecommendationPresenter where the view is passed in when it is created.